api/v1/framework/cypress.schema.json

```diff
@@ -70,6 +70,7 @@
           "$ref": "../subschema/common.schema.json#/definitions/version",
           "enum": [
             "package.json",
+            "10.3.1",
             "10.1.0"
```

Comment:
`10.1.0` was never released. Mind removing that one?

Reply:
sure. removing